packages/web/src/components/tipping/tip-audio/ConfirmSendTip.tsx

```diff
@@ -70,48 +62,50 @@ export const ConfirmSendTip = () => {
     </>
   )
 
-  const renderProfilePicture = () =>
-    profile ? (
-      <div className={styles.profileUser}>
-        <div className={styles.accountWrapper}>
-          <img className={styles.dynamicPhoto} src={profileImage} />
-          <div className={styles.userInfoWrapper}>
-            <div className={styles.name}>
-              {profile.name}
-              <UserBadges
-                userId={profile?.user_id}
-                badgeSize={12}
-                className={styles.badge}
-              />
-            </div>
-            <div className={styles.handleContainer}>
-              <span className={styles.handle}>{`@${profile.handle}`}</span>
-            </div>
-          </div>
-        </div>
+  const renderConfirmInfo = () => (
```

Comment:
one thought is that these can just be const like `confirmInfoElement` this is fine tho